ansible/roles/slapd/defaults/main.yml

```diff
@@ -737,6 +737,7 @@ slapd__default_tasks:
     state: 'exact'
 
     # Warning: changes in this attribute require slapd restart
+    # (fixed for slapd 2.5, see https://bugs.openldap.org/show_bug.cgi?id=6035)
```

Comment:
This comment isn't really useful here - what exactly was fixed? We have to go to the URL to see that the restart for changes in `olcAuthzRegexp` attribute is not needed on `slapd` 2.5. That will be great, when `slapd` 2.5 is finally released. :-)

I suppose that if you want to amend the warning, I would go with something like:
```yaml
# Warning: OpenLDAP v2.4.x requires a restart when this attribute is changed
```

Reply:
Ok, done and updated branch force-pushed